clmm/modbackend/func_layer.py

```diff
@@ -39,13 +39,13 @@ def get_3d_density(r3d, mdelta, cdelta, z_cl, cosmo, delta_mdef=200, halo_profil
     halo_profile_model : str, optional
         Profile model parameterization, with the following supported options:
             `nfw` (default)
-            `einasto`
-            `hernquist`
+            `einasto` (not in cluster_toolkit)
+            `hernquist` (not in cluster_toolkit)
```

Comment:
Actually, running the OO modeling demo with `ccl` and `einasto`, returns a CCL error, linked I think to the numerical integration in CCL (I remember @pennalima metionning there was a problem in CCL along those lines). So really, `einasto` is not supported by the CCL backend either.

Regarding the `hernquist` profile with CCL, the demo notebook runs but I saw significant numerical instabilities in the plots, so I'm not sure it is properly supported either. Maybe @pennalima or @vitenti  can confirm?

Reply:
Yes, their default settings give very poor results, I would mark anything other than NFW as 'unstable' under the CCL backend.

Reply:
Actually, analytic NFW!!!
Numerical NFW also "fails".

Reply:
So should we explicitly prevent the user from trying those keywords with CCL, or simply add a very loud warning?

Reply:
I think this will be fixed in the next CCL release. So we can include a huge warning: if CCL v?.? use only analytic NFW.

Reply:
@combet @pennalima @vitenti 
I added this check. I set CCL benchmark version to 10 while we wait for a working one.